### DISCUSSION TITLE
Z Axis PROBE Offset

### DISCUSSION THREAD

**Discussion by tracetinkerer** [Jyers/Marlin#1479] [Q&A] [answered]
- Is it now baked into firmware?

**Comment by tracetinkerer** [comment, marked as answer]
I recall there being an option for me to plug in the offset for the X, Y and Z offsets so the machine knows the location of the nozzle vs the probe. Since Installing this firmware I cant find it, so I'm assuming that I no longer need to give the machine this information as it's specified in the firmware?

**Reply by tracetinkerer** [reply]
Hero! Thank you, it's been a while.

**Reply by MicroSur** [reply]
#define NOZZLE_TO_PROBE_OFFSET { -45, -12, 0 } // Jyers {  -45.5, -8, -2.2 }
Is the third parameter be Z probe offset in FW?
